src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -948,7 +948,14 @@ public void spawnCitizen(final CitizenData data)
             CitizenData citizenData = data;
             if (citizenData == null)
             {
-                topCitizenId++;
+                for (int i = 1; i <= 100; i++)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)